android/conf/single.conf.js

```diff
@@ -0,0 +1,18 @@
+USERNAME =  process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
+ACCESS_KEY = process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
+  
+exports.capabilities = {
+    "platformName" : "android",
+    "appium:deviceName" : "Google Pixel 6 Pro",
+    "browserName" : "chrome",
```

Comment:
do we need browserName caps ?

Reply:
sadly yes, to use selenium webdriver.

Reply:
Possible to explain ? Do we pass both app id and browser combined for bot appium  ?

Reply:
https://github.com/browserstack/mocha-appium-app-browserstack/pull/1/files#r1565167294